src/components/themed/WalletListCurrencyRow.js

```diff
@@ -184,14 +66,26 @@ export const WalletListCurrencyRowComponent = (props: Props) => {
       <View style={styles.nameColumn}>
         <View style={styles.currencyRow}>
           <EdgeText style={styles.currencyText}>{currencyCode}</EdgeText>
-          {showRate ? <EdgeText style={[styles.exchangeRateText, styles[exchangeRateType ?? 'neutral']]}>{exchangeRateText}</EdgeText> : null}
+          {showRate && wallet != null ? (
+            <TickerText currencyConfigMap={currencyConfigMap} wallet={wallet} tokenId={tokenId} style={styles.exchangeRateText} />
```

Comment:
This `<TickerText />` should be wrapped in an `<EdgeText style={styles.exchangeRateText}>` component, assuming we remove the style prop from `TickerText` itself.